src/modules/admin/components/users/ClientAccessSummaryTable.tsx

```diff
@@ -0,0 +1,64 @@
+import { omit } from 'lodash-es';
+import React from 'react';
+
+import { ColumnDef } from '@/components/elements/table/types';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import { parseAndFormatDateTime } from '@/modules/hmis/hmisUtil';
+import {
+  ClientAccessSummaryFieldsFragment,
+  GetUserClientSummariesDocument,
+  GetUserClientSummariesQuery,
+  GetUserClientSummariesQueryVariables,
+} from '@/types/gqlTypes';
+
+const columns: ColumnDef<ClientAccessSummaryFieldsFragment>[] = [
+  {
+    header: 'Client Name',
+    render: 'clientName',
+  },
+  {
+    header: 'Client ID',
+    render: 'clientId',
+  },
+  {
+    header: 'Last Accessed',
+    render: ({ lastAccessedAt }) => parseAndFormatDateTime(lastAccessedAt),
+  },
+];
+
+interface Props {
+  userId: string;
+  startDate?: string;
+  searchTerm?: string;
+}
+const ClientAccessSummaryTable: React.FC<Props> = ({
+  userId,
+  startDate,
+  searchTerm,
+}) => {
+  return (
+    <GenericTableWithData<
+      GetUserClientSummariesQuery,
+      GetUserClientSummariesQueryVariables,
+      ClientAccessSummaryFieldsFragment
+    >
+      queryVariables={{
+        id: userId,
+        filters: { searchTerm },
+      }}
+      queryDocument={GetUserClientSummariesDocument}
+      defaultFilters={{
+        onOrAfter: startDate,
+      }}
+      columns={columns}
+      pagePath='user.clientAccessSummaries'
+      noData='No access history'
+      filterInputType='ClientAccessSummaryFilterOptions'
+      recordType='access events'
```

Comment:
"events" feels sort of confusing since it's really a table of clients. I think this is a little clearer.
```suggestion
      recordType='clients accessed'
```

Reply:
Sure thing, I just added it since if recordType is undefined, the filters aren't displayed (which is kind of  unintuitive)